src/Command/ModelCommand.php

```diff
@@ -346,11 +346,18 @@ public function findBelongsTo(Table $model, array $associations): array
                 ];
             } else {
                 $tmpModelName = $this->_modelNameFromKey($fieldName);
-                if (!in_array(Inflector::tableize($tmpModelName), $this->_tables, true)) {
+                $associationTable = $this->getTableLocator()->get($tmpModelName);
+                $tables = $this->listAll();
```

Comment:
You should be able to use `$this->_tables` here as it was used here before.

Reply:
listAll() returns the value of $_tables if it's already populated else fetches the list from the db. I think using the method is fine here. That way it doesn't need to rely on listAll() being called before this method is invoked.

Reply:
That's correct. I initially used $this->_tables but found unit tests failed when findBelongsTo was called isolated. generateAssociations calls listAll as well as findHasMany and findBelongsToMany methods but findBelongsTo didn't.